### PR TITLE
To DSP guide, add latency recommendations

### PR DESCRIPTION
**Pull request by genwhittTTD**
To DSP guide, add latency recommendations

### CODE REVIEW

File: docs/guides/dsp-guide.md [hunk 1]
```diff
@@ -80,6 +80,15 @@ The following table provides details for Step 2 of the workflow diagram shown in
 | 2-a | Server-side SDK (see [SDKs: Summary](../sdks/summary-sdks.md)) | Leverage the provided SDK to decrypt incoming UID2 tokens. The response contains the `UID2` and the UID2 creation time. |
 | 2-b | | DSPs are required to honor opt-out protocol for UID2s. For details on configuring user opt-outs and honoring them during bidding, see [Honor user opt-outs](#honor-user-opt-outs). |
 
+## Recommendations for Managing Latency
+
+For a low latency/high throughput setup, follow these recommendations:
+
+- Have a local instance of the UID2 client for each server. This can be in-proc or out-of-proc. In-proc is easiest.
```
Comment (sunnywu) [inline docs/guides/dsp-guide.md:87]: let's use the proper term in-process/out-of-process

File: docs/guides/dsp-guide.md [hunk 2]
```diff
@@ -80,6 +80,15 @@ The following table provides details for Step 2 of the workflow diagram shown in
...
+- Call client refresh periodically in the background: for example, once per hour, with some randomization to avoid peaks after global fleet restarts.
+- When a token needs to be decrypted, call client decrypt. In-proc is fastest, but out-of-proc is also acceptable if it's done correctly. (**GWH_SW01 how do they know how to make sure it's done correctly?**)
```
Comment (sunnywu) [inline docs/guides/dsp-guide.md:89]: in/out of **process**

**Review by sunnywu — APPROVED** [review]

File: docs/guides/dsp-guide.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/guides/dsp-guide.md:87]: Done, thx.

File: docs/guides/dsp-guide.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/guides/dsp-guide.md:89]: Done, thx.

**Pull request merged**